src/web/mjs/connectors/DynastyScans.mjs

```diff
@@ -52,12 +55,13 @@ export default class DynastyScans extends Connector {
      *
      */
     _getChapterList( manga, callback ) {
-        this.fetchDOM( this.url + manga.id, '.chapter-list dd a[class="name"]' )
-            .then( data => {
-                let chapterList = data.map( element => {
-                    let title = element.text.replace( manga.title, '' ).trim();
+        this.fetchJSON( this.url + manga.id + '.json' )
+            .then( json => {
+                // filter removes the "header" elements from the list
+                let chapterList = json['taggings'].filter( chapter => !chapter.header ).map( chapter => {
+                    let title = chapter.title.replace( manga.title, '' ).trim();
```

Comment:
Does the json data still has the manga title within the chapter title and needs to be replaced?

Reply:
Maybe not, it's the same as in HTML pages. Actually I don't really remember why it was like that in the first place.

Reply:
Quickly searched but didn't find any example where this would apply.
However I found [an example](https://dynasty-scans.com/anthologies/an_absurd_relationship_collection) of why it might have been done this way. The same series is also under another format [here](https://dynasty-scans.com/series/an_absurd_relationship).
I'll simplify it since it can easily be restored later if needed anyway.